aeron-driver/src/main/java/io/aeron/driver/Configuration.java

```diff
@@ -1438,6 +1452,28 @@ public static long flowControlReceiverTimeoutNs()
             getDurationInNanos(MIN_FLOW_CONTROL_TIMEOUT_OLD_PROP_NAME, FLOW_CONTROL_RECEIVER_TIMEOUT_DEFAULT_NS));
     }
 
+    /**
+     * Retransmit receiver window multiple used by the unicast flow control strategy to determine the maximum
+     * amount of data to retransmit.
+     *
+     * @return multiple.
+     */
+    public static int flowControlUnicastRetransmitReceiverWindowMultiple()
+    {
+        return getInteger(FLOW_CONTROL_UNICAST_RETRANSMIT_RECEIVER_WINDOW_MULTIPLE_PROP_NAME, 16);
+    }
+
+    /**
+     * Retransmit receiver window multiple used by multicast flow control strategies to determine the maximum
+     * amount of data to retransmit.
+     *
+     * @return multiple.
+     */
+    public static int flowControlMulticastRetransmitReceiverWindowMultiple()
```

Comment:
I think to follow existing naming convention this should be `multicastFlowControlRetransmit....`